### DISCUSSION TITLE
markdown 문법을 인식하지 못하는 것 같습니다.

### DISCUSSION THREAD

**Discussion by DI-LEE** [pronist/hELLO#69] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제:  windows
- 브라우저: chrome
- 버전: [e.g. 22]
- 스킨 버전 : 3.6.1

**설명**
markdown 문법에서 목차와 인용문이 인식이 되지 않습니다.

**발생 과정**
예전의 포스팅 해놓은 글 뿐만 아니라 새로 포스팅한 글 모두 markdown 문법에서 목차와 인용문이 인식되지 않습니다.

**스크린샷 및 링크**

<img width="595" alt="image" src="https://user-images.githubusercontent.com/79498819/153701746-e03f8ac4-3f22-4990-a9a1-10216c04b11b.png">

https://dilee.tistory.com/2

감사합니다.

**Comment by pronist** [comment, marked as answer]
@DI-LEE 안녕하세요. **스킨 소개페이지 - 트러블 슈팅 - 더보기**에 나와있듯이 마크다운에 대해서는 현재 대응하지 않고있습니다.  현재는 일반인을 비롯한 많은 사람들이 사용하는 **기본 티스토리 에디터**에 대해서만 대응하고 있음을 알려드립니다. 마크다운 관련해서는 추후 패치가 될 지도 모릅니다만, 일단 지금은 그렇습니다. 이점 참고부탁드립니다.
